pkg/replicate/replicator.go

```diff
@@ -243,3 +235,24 @@ func RunReplicate(
 
 	return nil
 }
+
+func newMetaFetcher(logger log.Logger, fromBkt objstore.InstrumentedBucket, reg prometheus.Registerer, minTime, maxTime thanosmodel.TimeOrDurationValue, concurrency int, ignoreMarkedForDeletion bool) (*thanosblock.MetaFetcher, error) {
```

Comment:
Abstracted to separate func so it can be used in tests

Reply:
Thanos uses multiple lines for that many parameters, I think? Maybe you can add some line breaks. :relaxed:

Reply:
done!

Reply:
I don't get why this helps in testing. Can you please elaborate more?

Reply:
Well I don't say it's, much but at least it abstracts the creation of the fetcher based on the input flags config and the initialization and configuration of the fetcher filters is the same in the tests and real code path.

Reply:
I see. That sounds good.